### PR TITLE
Add support for empty var with None default value

### LINKED ISSUE #209: Support for SOME_VAR=(int, None)?
Is there a way to tell environ that `SOME_VAR=` in an env file is fine, and should lead to SOME_VAR being assigned the pythonic `None` when using the `SOME_VAR=(int, None)` format?

### PR DESCRIPTION
**Pull request by mehdy**
Closes #209

### CODE REVIEW

**Comment by mehdy** [comment]
@sergeyklay ready for review.

**Comment by sergeyklay** [comment]
@mehdy Could you please update change log?

**Comment by sergeyklay** [comment]
@mehdy Please note, I changed the base branch from `main`  to `develop`

**Comment by mehdy** [comment]
Done!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -16,6 +16,8 @@ Added
 - Added option to override existing variables with ``read_env``
   `#103 <https://github.com/joke2k/django-environ/issues/103>`_,
   `#249 <https://github.com/joke2k/django-environ/issues/249>`_.
+- Added support for empty var with None default value
+  `#332 <https://github.com/joke2k/django-environ/pull/332>`_.
```
Comment (sergeyklay) [inline CHANGELOG.rst:20]:
```suggestion
  `#209 <https://github.com/joke2k/django-environ/issues/209>`_.
```

Comment (mehdy) [reply CHANGELOG.rst]: Oh, I made the change!

**Pull request merged**